Testscripts/Windows/STOR-VSS-BackupRestore-ISO-NoNetwork.ps1

```diff
@@ -66,53 +66,45 @@ function Main {
         $defaultVhdPath = $obj.DefaultVirtualHardDiskPath
         if (-not $defaultVhdPath) {
             Write-LogErr "Unable to determine VhdDefaultPath on Hyper-V server ${hvServer}"
-            $error[0].Exception
+            $exception = $error[0].Exception.Message
+            Write-LogErr "$exception"
             return $False
         }
         if (-not $defaultVhdPath.EndsWith("\")) {
             $defaultVhdPath += "\"
         }
         $isoPath = $defaultVhdPath + "${vmName}_CDtest.iso"
-        Write-LogInfo "iso path: $isoPath defaultVhdPath $defaultVhdPath"
+        Write-LogInfo "ISO path: $isoPath defaultVhdPath $defaultVhdPath"
         $WebClient = New-Object System.Net.WebClient
         $WebClient.DownloadFile("$url", "$isoPath")
         try {
-            Get-RemoteFileInfo -filename $isoPath  -server $HvServer
-        }
-        catch {
+            Get-RemoteFileInfo -filename $isoPath -server $HvServer | Out-Null
+        } catch {
             Write-LogErr "The .iso file $isoPath could not be found!"
```

Comment:
no need to duplicate the error msg. if you add msg in throw statement, then no need to keep Write-LogErr.